pavelib/tests.py

```diff
@@ -212,6 +212,26 @@ def coverage(options):
     call_task('diff_coverage', options=dict(options))
 
 
+@no_help
+@task
+@needs('pavelib.prereqs.install_prereqs')
+def combine_jenkins_coverage():
+    """
+    Combine coverage reports.
+    """
+    coveragerc = Env.REPO_ROOT / 'test_root' / '.jenkins-coveragerc'
+
+    for directory in Env.LIB_TEST_DIRS + ['cms', 'lms']:
+        report_dir = Env.REPORT_DIR / directory
+        if report_dir.isdir():
```

Comment:
The case would be that you haven't run tests of a certain group.  When I was testing this locally, there were times when I just wanted to run the cms tests, but didn't want it to fail because `reports/common/lib/calc` didn't exist.